packages/react/src/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -571,13 +716,13 @@ class OverflowMenu extends Component {
       <ClickListener onClickOutside={this.handleClickOutside}>
         <span
           className={`${prefix}--overflow-menu__wrapper`}
-          aria-owns={open ? menuBodyId : null}>
+          aria-owns={open ? menuBodyId : undefined}>
           <IconButton
             {...other}
             type="button"
             aria-haspopup
             aria-expanded={open}
-            aria-controls={open ? menuBodyId : null}
+            aria-controls={open ? menuBodyId : ''}
```

Comment:
Could this be `undefined` instead? It's adding `aria-controls=""` to the datatable snapshot, which shouldn't happen I think